tests/utest

```diff
@@ -137,6 +137,7 @@ run_utests() {
       std_base)
         CREATE_BASELINE=true
         RTPWD=$baseline_location
+        INPUTDATA_ROOT=$RTPWD
```

Comment:
I don't think you need this everywhere for every case, since it is defined on line 730.

Reply:
utest can be updated later to separate input data from baseline, but currently we use it for CI test only for fv3 control case, a canned case with INPUT data is all it needs. so CI test is OK to set as RTPWD.